src/app/components/Embeds/OEmbed/index.tsx

```diff
@@ -16,19 +20,53 @@ type OEmbedData = Partial<{
   width: number;
   height: number;
   type: string;
+  parameters: ampParams;
+  oEmbedType: string;
 }>;
 
 export type OEmbedProps = {
   type: string;
   oembed: OEmbedData;
 };
 
+export type ampParams = {
+  'amp-clickable': boolean;
+  'amp-image-height': number;
+  'amp-image-width': number;
+  'amp-image': string;
+};
+
+export type OEmbedAmpProps = {
+  isVDJEmbed: boolean;
+  canonicalLink: string;
+  parameters?: ampParams;
+  url?: string;
+};
+
 const OEmbedLoader = ({ oembed }: OEmbedProps) => {
-  const { translations } = useContext(ServiceContext);
   const { isAmp, canonicalLink } = useContext(RequestContext);
-  const { html } = oembed;
+  const { translations } = useContext(ServiceContext);
+  const { html, oEmbedType, parameters, url } = oembed;
+  const isVDJEmbed = oEmbedType === 'vdj-embed';
 
   if (isAmp) {
+    if (isVDJEmbed) {
+      if (parameters) {
+        const ampMetadata = {
+          imageWidth: parameters['amp-image-width']?.toString(),
+          imageHeight: parameters['amp-image-height']?.toString(),
```

Comment:
Should convert these values to String in the BFF? (It only recently occured to me)

The legacy VjAmp renderer requires these values to be strings.

Reply:
Looking at the AMP spec, it looks like these fields can be either numbers/strings: https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes#heights as numbers can be used or percentage based strings.

I'd almost be inclined to remove `.toString()` and remove the `string` requirement from the `VjAmp` component as both values for valid. I guess forcing `string` allows both types anyway, so its much of a muchness.

Reply:
Looking at the AMP spec: https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes#width-and-height it looks like these should really be numbers rather than strings.

I'd be inclined to remove `.toString()` and the `string` requirement from the `VjAmp` component as it doesn't seem entirely correct.

Reply:
Cool yep this makes sense - I have updated this